backend/functions/src/email/routes.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express'
+import { config } from 'dotenv'
+import { sendEmail } from './functions'
+
+const router = Router()
+config()
+
+// Modulated sending email functions
+router.post('/send', (req, res, next) => {
+  sendEmail(req, res)
+})
```

Comment:
From what I see, the basic structure here in the other parts of the backend is something like
1. extract values from `req.body`
2. call function with the values
3. use returned promise to `res.status`